test/12_nullablearray.jl

```diff
@@ -970,4 +970,27 @@ for ordered in (false, true)
     end
 end
 
+# Test vcat with nulls
```

Comment:
Please test `levels` and `isordered` on the result. That's really an essential property of categorical arrays.